pkg/oc/cli/cmd/wrappers.go

```diff
@@ -736,10 +736,9 @@ var (
 // NewCmdConfig is a wrapper for the Kubernetes cli config command
 func NewCmdConfig(parentName, name string, out, errOut io.Writer) *cobra.Command {
 	pathOptions := &kclientcmd.PathOptions{
-		GlobalFile:       cmdconfig.RecommendedHomeFile,
-		EnvVar:           cmdconfig.OpenShiftConfigPathEnvVar,
-		ExplicitFileFlag: cmdconfig.OpenShiftConfigFlagName,
-
+		GlobalFile:        cmdconfig.RecommendedHomeFile,
+		EnvVar:            cmdconfig.OpenShiftConfigPathEnvVar,
+		ExplicitFileFlag:  kclientcmd.RecommendedConfigPathFlag,
```

Comment:
Looks breaking.  Patches required?

Reply:
Breaking as in, this will be the default flag asking for kubeconfig. Isn't that what we need to to if we want to deprecate `--config` in 3.10?

Reply:
> Breaking as in, this will be the default flag asking for kubeconfig. Isn't that what we need to to if we want to deprecate --config in 3.10?

Does it respect `--config` at all anymore?